src/stores/messages/MessagesStore.ts

```diff
@@ -49,6 +50,12 @@ export default class MessagesStore {
 
 	public dataStore = new MessagesDataProviderStore(this, this.api);
 
+	@observable
```

Comment:
let's make MessagesExportStore or something like that to keep MessagesStore as thin as possible